infra/gcp/lib_util.sh

```diff
@@ -67,3 +67,19 @@ fi
 function indent() {
     ${SED} -u 's/^/  /'
 }
+
+# Join things with separator
+# Arguments:
+#   $1:  Separator
```

Comment:
comment that this must be a single character

Reply:
Done